applications/sam2/Dockerfile

```diff
@@ -16,40 +16,60 @@
 # limitations under the License.
 
 
-FROM nvcr.io/nvidia/pytorch:24.06-py3 as pytorch
+FROM nvcr.io/nvidia/pytorch:24.06-py3 AS pytorch
+# FROM nvcr.io/nvidia/pytorch:24.05-py3 as pytorch
 
 
 # Install dependencies
 RUN apt-get update && apt-get install -y \
-    git \
+    git \   
     git-lfs \ 
     x11-apps \
     && rm -rf /var/lib/apt/lists/*
 
 # Set up Git LFS
 RUN git lfs install
 
+
 # Clone sam2
 WORKDIR /workspace
 ARG COMPUTE_CAPACITY
-RUN git clone https://github.com/facebookresearch/segment-anything-2.git \
-    && cd segment-anything-2 \
+RUN git clone https://github.com/facebookresearch/sam2.git \
+    && cd sam2 \
     && python3 -m pip install --no-cache-dir -e . \
     && python3 -m pip install --no-cache-dir -e ".[demo]" \
     && cd checkpoints \
     && ./download_ckpts.sh
+# Add sam2 to PYTHONPATH
+ENV PYTHONPATH="/workspace/sam2"
```

Comment:
To confirm, this is overwritten by `dev_container` right? Prefer to keep this line for reference?

Reply:
I removed the PYTHONPATH variable alltogether from the Dockerfile. Instead the run.sh contains this part.